lib/schemas/cancel-order.js

```diff
@@ -14,7 +14,7 @@ const cancelOrderSchema = yup.object().shape({
     yup.object({
       supplier_id: yup.string().required(),
       fulfillment: yup.object().shape({
-        member_paid: amountAndCurrencyCode,
+        member_paid: yup.array().of(yup.object({ amount: yup.number(), currency_code: yup.string() })),
```

Comment:
Question: are `amount` and `currency_code` required by Pangea?

If so, can we add tests + update the schema accordingly?

Reply:
never mind, read the description